app/assets/src/components/common/DetailsSidebar/TaxonDetailsMode/taxon_details_mode.scss

```diff
@@ -67,3 +67,19 @@
     cursor: pointer;
   }
 }
+
+.wikiLicense {
+  padding-top: 10px;
+  color: $medium-grey;
+  font-size: $font-size-small;
```

Comment:
Can we define a font mixin?
@happyimadesignr Do we have a whole font style for the small attribution text?

Reply:
@tfrcarvalho I'd use this for any attribution or caption type text. but i dont know yet where else this will be

@cdebourcy @MarkAZhang I thought our naming for this font size was $font-body-xxxs?

Reply:
This file still references the old `styles/_typography` and not the new one `styles/themes/_typography`. Need to make more explicit that the previous one is deprecated, but since this file was already using the previous on, I guess this is fine.